x-pack/filebeat/module/imperva/securesphere/test/generated.log-expected.json

```diff
@@ -0,0 +1,5693 @@
+[
+    {
+        "destination.ip": [
+            "10.70.155.35"
+        ],
+        "destination.port": 892,
+        "event.action": "cancel",
+        "event.category": "Login",
+        "event.code": "Imperva",
+        "event.dataset": "imperva.securesphere",
+        "event.module": "imperva",
+        "event.original": "%IMPERVA-Imperva,dstIP=10.70.155.35,dstPort=892,dbUsername=tatno,srcIP=10.81.122.126,srcPort=4141,creatTime=29 January 2016 06:09:59,srvGroup=uam,service=untutl,appName=rad,event#=taliqu,eventType=Login,usrGroup=ommod,usrAuth=True,application=\"scivel\",osUsername=aqui,srcHost=radipis5408.mail.local,dbName=enatuse,schemaName=magn,bindVar=equuntu,sqlError=failure,respSize=5910,respTime=10.347000,affRows=sum,action=\"cancel\",rawQuery=\"sit\"",
```

Comment:
is `usrGroup` the group associated with the operating system user? If so, wondering if we can translate that to the ECS `group` fields